src/player.h

```diff
@@ -166,6 +166,11 @@ class Player
 
 	std::string inventory_formspec;
 
+	bool custom_deathscreen;
+	bool enable_deathscreen;
+	std::string get_deathscreen_formspec() { return deathscreen_formspec; }
```

Comment:
this function should be const. Also please attach deathscreen to SAO not Player we are trying to reduce/remove the Player class server side